packages/context/src/__tests__/unit/context.unit.ts

```diff
@@ -342,6 +343,86 @@ describe('Context', () => {
     });
   });
 
+  describe('findOrCreateBinding', () => {
+    describe('bindingCreationPolicy - ALWAYS_CREATE', () => {
```

Comment:
I think we usually use `context` instead of `describe` to group tests of the same subject but under different conditions.

```suggestion
    context('with ALWAYS_CREATE', () => {
```

This way the tests read like:

- findOrCreateBinding with ALWAYS_CREATE creates a new binding even the key is bound
- findOrCreateBinding with ALWAYS_CREATE creates a new binding if not bound
- ...